chaos_genius/core/utils/data_loader.py

```diff
@@ -121,14 +119,18 @@ def _convert_date_to_string(self, date: date, offset: str):
         # we shouldn't need to take offset as a string, but rather
         # take in a pytz timezone and skip using strings.
         date = date.strftime("%Y-%m-%d")
```

Comment:
Why isn't this parameterized too?

Reply:
While we could parameterize it too, I plan to refactor this entire function as soon as we deprecate 3 letter timezones to not use strings at all.